orc8r/cloud/docker/controller/supervisord.conf

```diff
@@ -165,6 +165,14 @@ stdout_events_enabled=true
 stderr_events_enabled=true
 
 # LTE services
+[program:had]
```

Comment:
The services are (well, were as of a month or two ago) sorted such that the "module" service is first, then after that they're alphabetized. If you wanna move this down beneath lte that would make me happy